### PR TITLE
Add Posichain networks (mainnet, testnet, devnet)

### PR DESCRIPTION
**Pull request by dannyposi**
Logo pinned: https://gateway.pinata.cloud/ipfs/QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8

### CODE REVIEW

**Comment by ligi** [comment]
> Logo pinned: https://gateway.pinata.cloud/ipfs/QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8

does not load for me - does it for you?

**Comment by dannyposi** [comment]
@ligi it look good for me, your first time maybe take longer than me. Could you have any suggest to pin it?

**Comment by dannyposi** [comment]
@ligi i've added logo to `_data/iconsDownload`. Is it ok?

**Review by ligi — APPROVED** [review]

File: _data/icons/posichain.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8",
+    "width": 300,
```
Comment (ligi) [inline _data/icons/posichain.json:4]: getting a different size here:

identify QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8                             12:54:55
QmXqprM1TPafQFLirQ7dMPMgou1hoUGuCvHYSWU9g8ocv8 SVG 24x24 24x24+0+0 8-bit sRGB 1018B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/posichain.json (hunk 1 above)
Comment (dannyposi) [reply _data/icons/posichain.json]: Thank, i've updated

**Review by ligi — APPROVED** [review]

**Pull request merged**